data/json/field_type.json

```diff
@@ -1229,8 +1229,8 @@
     "type": "field_type",
     "//": "1 < 10mm (0.39\"); 2 ≈ 10mm-160mm (6.3\");  3 ≈ 160mm-810mm (31.9\"); see map::actualize_snow_in_submap for details.",
     "intensity_levels": [
-      { "name": "shallow snow", "sym": "*", "color": "white", "convection_temperature_mod": -2 },
-      { "name": { "ctxt": "noun", "str": "snow" }, "color": "white" },
+      { "name": "shallow snow", "color": "light_gray", "convection_temperature_mod": -2 },
+      { "name": "snow" },
```

Comment:
Why removing the context though?

Reply:
Removed it while fiddling with the field definition and forgot to put it back. Thanks for spotting.